doc/source/rest.j2

```diff
@@ -551,15 +551,86 @@ argument, and in this case the second argument passed is the value, or it.
 The operators or (`or` or `∨`), and (`and` or `∧`) and `not` are also
 supported, and take a list of arguments as parameters.
 
+.. _aggregates:
+
+Aggregates: on the fly, measurements modification and aggregation
+===================================================================
+
+Gnocchi allows to do on-the-fly aggregation and modification of already
+aggregated data of |metrics|.
+
+It can be done by providing the list of |metrics| to aggregate:
+
+{{ scenarios['get-aggregates-by-metric-ids']['doc'] }}
+
+This example computes the mean aggregates with `all` metrics listed in
+`metrics` and then multiples it by `4`.
+
+.. Note:
+
+   `operations` can also be passed as a string, for example:
+   `"operations": "(aggregate mean (metric all))"`
+
+Operations between metrics can also be done, such as:
+
+{{ scenarios['get-aggregates-between-metrics']['doc'] }}
+
+Partial list of supported maths/aggregation::
+
+    ```
+    (rolling mean 2 (...))
+    (aggregate min (...))
+    (resample sum 60 (...))
+    (absolute (...))
+    (negative (...))
+    (>= (...) 2)
+    (* (...) 5)
+    (mod (...) 2)
+    ```
+
+Aggregation across |metrics| have different behavior depending
+on whether boundary values are set (`start` and `stop`) and if `needed_overlap`
+is set.
+
+Gnocchi expects that time series have a certain percentage of timestamps in
+common. This percent is controlled by the `needed_overlap` needed_overlap,
+which by default expects 100% overlap. If this percentage is not reached, an
+error is returned.
+
+.. note::
+
+   If a `start` and `stop` boundary is not set, Gnocchi will set the missing
+   boundary to the first or last timestamp common across all series.
+
+The ability to fill in points missing from a subset of time series is supported
+by specifying a `fill` value. Valid fill values include any float or `null`
+which will compute aggregation with only the points that exist. The `fill`
```

Comment:
This is not clear what "which" is refering to. You mean in case of `null`? if that's the case write:

> Valid fill values include any float or `null`. In the case of `null`, Gnocchi will compute the aggregation using only the existing points.

Reply:
done